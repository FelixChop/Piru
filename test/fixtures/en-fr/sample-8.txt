The hermit lived in austere conditions, but claimed that simplicity was the spice of life.
