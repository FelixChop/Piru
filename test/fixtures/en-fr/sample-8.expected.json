[
  {
    "word": "hermit",
    "lemma": "hermit",
    "definition": "ermite",
    "citation": "The hermit lived in austere conditions, but claimed that simplicity was the spice of life.",
    "to_learn": true
  },
  {
    "word": "austere",
    "lemma": "austere",
    "definition": "simple, austère",
    "citation": "The hermit lived in austere conditions, but claimed that simplicity was the spice of life.",
    "to_learn": true
  },
  {
    "word": "spice of life",
    "lemma": "spice of life",
    "definition": "ce qui rend la vie intéressante",
    "citation": "The hermit lived in austere conditions, but claimed that simplicity was the spice of life.",
    "to_learn": true
  }
]
